testdata/filters/attributegen.yaml.tmpl

```diff
@@ -0,0 +1,29 @@
+- name: istio.attributegen
+  typed_config:
+    "@type": type.googleapis.com/udpa.type.v1.TypedStruct
+    type_url: type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
+    value:
+      config:
+        vm_config:
+          runtime: {{ .Vars.WasmRuntime }}
+          vm_id: stats_outbound{{ .N }}
+          code:
+            local: { inline_string: "envoy.wasm.attributegen" }
+        configuration:
+          attributes:
```

Comment:
You will need to add `@type` url to the configurations, like: https://github.com/istio/proxy/blob/9bbf80d5397880a8a1a4c3c8e4c36c2ff2575e30/testdata/filters/stats_inbound.yaml.tmpl#L14

This is something new and does not need in 1.6.

Reply:
In case you did not notice it, you will also need to move the configurations under a value block as well:
https://github.com/istio/proxy/blob/9bbf80d5397880a8a1a4c3c8e4c36c2ff2575e30/testdata/filters/stats_inbound.yaml.tmpl#L15

Like this change: https://github.com/istio/proxy/pull/2862/files#diff-9aa8081ed5cd5469d42889f40a0ba32e